### PR TITLE
added pyemma to topical software, section MD

### PR DESCRIPTION
**Pull request by marscher**
There was a faulty entry in the optimization section stating PyEMMA (referencing our homepage) is a software for constrained programming, which is not true. The optimization software mentioned is on pypi, but without a source code or binary. So I removed it. The homepage of the package is also not reachable http://www.eveutilities.com/products/emma

This PR adds PyEMMA to the molecular modelling section.

cc @franknoe

### CODE REVIEW

File: www/topical-software.rst [hunk 1]
```diff
@@ -422,6 +420,7 @@ Molecular modeling
...
+- _`PyEMMA <https://emma-project.org/>`__ (EMMA = Emma's Markov Model Algorithms) is an open source Python/C package for analysis of extensive molecular dynamics simulations. In particular, it includes algorithms for estimation, validation and analysis of Markov state models, a popular toolset to gain insight on the kinetics of the simulation. It provides collective variables calculation, clustering methods, time lagged independent component analysis (TICA) and model estimation for Markov state models, hidden Markov models and multi ensemble simulation like umbrella sampling. In addition transition path theory can be applied onto these models, which allows for kinetic pathway extraction and flux computations via transition networks.
```
Comment (rgommers) [inline www/topical-software.rst]: The `_` before the link should be removed, gives a build warning now. 

Other than that these changes look good to me.

Comment (marscher) [reply www/topical-software.rst]: Thanks - I changed that.

**Comment by rgommers** [comment]
Merged, thanks @marscher

**Comment by rgommers** [comment]
@marscher I just noticed that https://emma-project.org has an invalid certificate, you may want to fix that.

**Comment by marscher** [comment]
I will notify our IT department about that, thanks again.

**Pull request merged**